uptimer-server/src/graphql/resolvers/user.ts

```diff
@@ -29,6 +37,52 @@ export const UserResolver = {
         password,
       } as IUserDocument;
       const result: IUserDocument | undefined = await createNewUser(authData);
+      const response: IUserResponse = await userReturnValue(
```

Comment:
Fix - Argument of type 'Request<ParamsDictionary, any, any, ParsedQs, Record<string, any>>' is not assignable to parameter of type 'Request'.
  Type 'Request<ParamsDictionary, any, any, ParsedQs, Record<string, any>>' is missing the following properties from type 'Request': cache, credentials, destination, integrity, and 13 more.